routes/web.php

```diff
@@ -23,6 +24,9 @@
 Route::get('/photo/all', 'PhotoController@all');
 Route::post('/photo/save', 'PhotoController@savePhoto');
 
+Route::get('/token', 'TokenController@index');
+Route::post('/token', 'TokenController@addToken')->middleware('permissions:admin');
```

Comment:
could we register middleware in controller?

Reply:
Why do you want register middleware in controller? As we discussed before you wanted to put all middleware logic to routes.